jumpscale/packages/vdc_dashboard/chats/mattermost.py

```diff
@@ -19,30 +21,28 @@ class MattermostDeploy(SolutionsChatflowDeploy):
         {"cpu": 10, "memory": 10},  # initContainer.remove-lost-found
     ]
 
+    def get_config(self):
+        return {
+            "ingress.host": self.config.chart_config.domain,
+            "mysql.mysqlUser": self.config.chart_config.mysql_user.value,
+            "mysql.mysqlPassword": self.config.chart_config.mysql_password.value,
+            "mysql.mysqlRootPassword": self.config.chart_config.mysql_root_password.value,
+        }
+
     @chatflow_step(title="Configurations")
     def set_config(self):
 
         form = self.new_form()
-        mysql_user = form.string_ask("Enter mysql user name", default="mysql", min_length=3, required=True,)
-        mysql_password = form.secret_ask(
+        self.config.chart_config.mysql_user = form.string_ask(
+            "Enter mysql user name", default="mysql", min_length=3, required=True,
+        )
+        self.config.chart_config.mysql_password = form.secret_ask(
             "Enter mysql password", default="mySqlPassword", min_length=8, required=True,
         )  # TODO: need to check a valid password
-        mysql_root_password = form.secret_ask(
+        self.config.chart_config.mysql_root_password = form.secret_ask(
```

Comment:
we need to set .value out of form.ask, because inside it'll be a for chatflow object